### ISSUE TITLE
Add slight color formatting between Tweets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [emuikernel/wittytwitter#12]
```
What steps will reproduce the problem?
1. Open up Witty


What is the expected output? What do you see instead?
I would like to see a slight shading difference between the tweets.  That 
allows for a better view.

What version of the product are you using? On what operating system?
0.1.3



```

Original issue reported on code.google.com by `kurtbroc...@gmail.com` on 19 Oct 2007 at 3:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `alan...@gmail.com` on 19 Oct 2007 at 5:13
- Changed state: **Accepted**
- Added labels: **Milestone-v0.1.4**

**Comment by GoogleCodeExporter** [comment]
```
fixed in v0.1.4
```

Original comment by `alan...@gmail.com` on 20 Oct 2007 at 11:59
- Changed state: **Fixed**
